Cargo.toml

```diff
@@ -102,3 +100,6 @@ required-features = ["libc", "shm"]
 [[example]]
 name = "xeyes"
 required-features = ["shape"]
+
+[workspace]
+members = ["generator", "xcbgen-rs"]
```

Comment:
> Oh, the docs even start with explaining this kind of workspaces. x11rb is now the "root package" of the workspace. So, I guess this is fine and should be properly supported by cargo.

Exactly, cargo will remove the `[workspace]` section when publishing on crates.io.

> I guess the plan should be to have `xcbgen-rs` on `crates.io` and `generator` not, right?

Indeed, `xcbgen-rs` might be useful for others, so it could be eventually published. `generator` is very specific, so it does not make sense to publish it.

For now, both have `publish = false` to avoid accidentally publishing them.